src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.UnitConverter/InputInterpreter.cs

```diff
@@ -168,6 +168,22 @@ public static void FeetToFt(ref string[] split)
             }
         }
 
+        /// <summary>
+        /// Converts spelling "metre" to "meter"
+        /// </summary>
+        public static void MetreToMeter(ref string[] split)
+        {
+            if (split[1].ToLower() == "metre")
```

Comment:
should we verify `split` array bounds?

Reply:
They are already verified in calling method.
check in Parse method
`            if (split.Length != 4)
            {
                // deny any other queries than:
                // 10 ft in cm
                // 10 ft to cm
                return null;
            }`

Reply:
is metre an translated version of meter? - Then this should be fixed by adding translation for the units

Reply:
I agree but there are no localization yet.

Reply:
yes. no one did that for unit in the past.